src/components/Comments.jsx

```diff
@@ -0,0 +1,27 @@
+import getComments from "../utils/getComments";
+import { useState ,useEffect } from "react";
+import Loading from "./Loading";
+import { useParams } from 'react-router-dom'
+import formatDate from "../utils/FormatDates";
+
+const CommentSection = () => {
+    const [comments,setComments] = useState([]);
+    const  {article_id} = useParams();
+
+    useEffect(() => {
+        getComments(setComments, article_id * 1);
+    },[]);
+
+    return <Loading>
```

Comment:
the loading should depend on a state which is set to true or false depending on when the loading of comments completes. at the moment the loading component checks whether there are any children and if there arent, e.g. if there are no comments on an article, it is in a permanent state of loading

Reply:
this feedback hasnt been implemented yet - the loading component will always render if there are no comments